includes/class-wc-validation.php

```diff
@@ -79,6 +79,9 @@ public static function is_postcode( $postcode, $country ) {
 			case 'IE':
 				$valid = (bool) preg_match( '/([AC-FHKNPRTV-Y]\d{2}|D6W)[0-9AC-FHKNPRTV-Y]{4}/', wc_normalize_postcode( $postcode ) );
 				break;
+			case 'IN':
+				$valid = (bool) preg_match( '/^([0-9]{6})$/', $postcode );
```

Comment:
This regex doesn't provide proper validation for postcodes. Since postcodes in India never suppose to start with `0`, also it suppose to allow an space too.

Will work better using `^[1-9]{1}[0-9]{2}\s{0,1}[0-9]{3}$`.

Reply:
Thanks. You are right. Pin Code in India do NOT start with zero.
However, space is not required. It is just a writing quirk that has crept into common usage. All systems (like Woocommerce) should not encourage quirks but move towards standardization.

So final regex should be `/^([1-9]{6})$/`

Thank you for reviewing this. Will help me gain confidence on working on PRs

Reply:
Sorry, I am not  sure how to proceed now to change the regex to `/^([1-9]{6})$/`
Do I need to create a fresh  PR?

Reply:
Seems like you excluded your fork, that's why you can't update.